sub/sd_ass.c

```diff
@@ -450,8 +450,16 @@ static void configure_ass(struct sd *sd, struct mp_osd_res *dim,
     ass_set_hinting(priv, set_hinting);
     ass_set_line_spacing(priv, set_line_spacing);
 #if LIBASS_VERSION >= 0x01600010
-    if (converted)
+    if (converted) {
         ass_track_set_feature(track, ASS_FEATURE_WRAP_UNICODE, 1);
+        if (!opts->sub_vsfilter_bidi_compat) {
+            for (int n = 0; n < track->n_styles; n++) {
+                track->styles[n].Encoding = -1;
+            }
```

Comment:
```suggestion
            for (int n = 0; n < track->n_styles; n++)
                track->styles[n].Encoding = -1;
```

Is it safe to override with -1 always?

Reply:
Since it's only for converted subtitles, I think so. Either way it can be disabled via the option if it has adverse effects but that should never happen.